rustls/src/server/builder.rs

```diff
@@ -101,7 +112,14 @@ impl ConfigBuilder<ServerConfig, WantsServerCert> {
             .provider
             .key_provider
             .load_private_key(key_der)?;
+
         let certified_key = CertifiedKey::new(cert_chain, private_key);
+        match certified_key.keys_match() {
```

Comment:
I think this is a better way of spelling it:

```rust
match certified_key.keys_match() {
    // Don't treat unknown consistency as an error
    Ok(()) | Err(Error::InconsistentKeys(InconsistentKeys::Unknown)) => (),
    Err(err) => return Err(err),
}
```

Reply:
Done! And agreed—this makes the intention and behavior more obvious